backend/app/chat/schemas.py

```diff
@@ -48,7 +48,7 @@ class ChatResponse(BaseModel):
         examples=["This is a sample chat response"],
     )
     response_metadata: Optional[dict] = Field(
-        default_factory=dict, examples=[{"timestamp": "2024-10-16T12:31:00Z"}]
+        default_factory=lambda: {}, examples=[{"timestamp": "2024-10-16T12:31:00Z"}]
```

Comment:
Hmmm... i thought I fixed this in my PR.

Reply:
Yeah I can't find it in codebase any more - should go away when I update to `develop`

Reply:
Sorted with a rebase 🫡